packages/salesforcedx-vscode-core/test/jest/commands/util/data/testData.ts

```diff
@@ -6,4 +6,4 @@
  */
 
 export const dummyOutputPull =
-  '{\n  "status": 0,\n  "result": {\n    "pulledSource": [\n      {\n        "state": "Created",\n        "fullName": "F10",\n        "type": "ApexClass",\n        "filePath": "/Users/kenneth.lewis/scratchpad/NewProj1/force-app/main/default/classes/F10.cls"\n      },\n      {\n        "state": "Created",\n        "fullName": "F10",\n        "type": "ApexClass",\n        "filePath": "/Users/kenneth.lewis/scratchpad/NewProj1/force-app/main/default/classes/F10.cls-meta.xml"\n      },\n      {\n        "state": "Changed",\n        "fullName": "Admin",\n        "type": "Profile",\n        "filePath": "/Users/kenneth.lewis/scratchpad/NewProj1/force-app/main/default/profiles/Admin.profile-meta.xml"\n      }\n    ]\n  },\n  "warnings": [\n    "We plan to deprecate this command in the future. Try using the \\"project retrieve start\\" command instead.",\n    "The loglevel flag is no longer in use on this command. You may use it without error, but it will be ignored.\\nSet the log level using the `SFDX_LOG_LEVEL` environment variable."\n  ]\n}\n';
+  '{\n  "status": 0,\n  "result": {\n    "files": [\n      {\n        "state": "Created",\n        "fullName": "F10",\n        "type": "ApexClass",\n        "filePath": "/Users/kenneth.lewis/scratchpad/NewProj1/force-app/main/default/classes/F10.cls"\n      },\n      {\n        "state": "Created",\n        "fullName": "F10",\n        "type": "ApexClass",\n        "filePath": "/Users/kenneth.lewis/scratchpad/NewProj1/force-app/main/default/classes/F10.cls-meta.xml"\n      },\n      {\n        "state": "Changed",\n        "fullName": "Admin",\n        "type": "Profile",\n        "filePath": "/Users/kenneth.lewis/scratchpad/NewProj1/force-app/main/default/profiles/Admin.profile-meta.xml"\n      }\n    ]\n  },\n  "warnings": [\n    "We plan to deprecate this command in the future. Try using the \\"project retrieve start\\" command instead.",\n    "The loglevel flag is no longer in use on this command. You may use it without error, but it will be ignored.\\nSet the log level using the `SFDX_LOG_LEVEL` environment variable."\n  ]\n}\n';
```

Comment:
The message says `"We plan to deprecate this command in the future. Try using the \\"project retrieve start\\" command instead."`  So it's outdated and no longer applies, right?